### ISSUE TITLE
Open Liberty SSL Database Connections Primer

### ISSUE THREAD

**Issue by KyleAure** [OpenLiberty/blogs#1270]
# Sample Outline

- Intro: What are SSL database connections and why are they important?
- How do database drivers establish SSL connections? 
- Common database primers
  - DB2
  - Postgres
  - Oracle
  - SQLServer
- How to setup other database drivers?  

# Release date
TBD

**Comment by gljrobinson** [comment]
Hi @KyleAure thanks for contributing this blog idea and creating this issue for it. 
It sounds really interesting. Is the plan to take an Open Liberty specific stance on all of these? Do you have a draft at all available?

**Comment by KyleAure** [comment]
Hello @GraceJansen ,
Thanks for the follow up.  In general Open Liberty supports the use of SSL Database Connections.  However, since the JDBC spec does not have a standard way to configure / create SSL connections each database has different config elements and methods of enabling the TLS protocol. The purpose of this blog would be to give customers a staring off point (sample configurations) and an explanation of how the main database drivers we support need to be configured to enable TLS.  

I plan to have a draft completed next week.

**Comment by gljrobinson** [comment]
Hi @KyleAure, just wanted to check in and see if there was a draft ready for our editors to take a look at for this blog?

**Comment by gljrobinson** [comment]
Hi @KyleAure, would you mind giving us an update as to when a first draft of this is planned for?

**Comment by KyleAure** [comment]
Hello @GraceJansen sorry for the long wait on this.  Got interrupted.  I posted a draft under PR #1404 
Let me know if I did that correctly.
Thank Kyle

**Comment by gljrobinson** [comment]
Hey @KyleAure, no worries, thanks for creating this blog. I just took a look at your PR and noticed that you're trying to merge into prod. Can I ask, have you merged into our draft branch yet?
If you have already merged into draft and checked how it looks on our staging blog site (https://blogs-draft-openlibertyio.mybluemix.net/blog/) then please would you close this PR to prod and create a PR to staging? This is where our editors take a look at it and help tweak or edit them before we do any PRs to prod. 
Thanks 

@dmuelle FYI

**Comment by KyleAure** [comment]
Hello @GraceJansen okay sorry for confusion.  I have opened a PR into the draft branch here under #1421
